examples/custom_search_method/core_search_runner/README.md

```diff
@@ -0,0 +1,36 @@
+# Custom SearchMethod with CoreSearchRunner
+
+This example shows how to implement a custom hyperparameter SearchMethod that enables fault tolerance.
+To run the custom SearchMethod, in this example we use CoreSearchRunner that executes the custom SearchMethod
+in the Determined cluster. 
+For an example of running the custom SearchMethod locally, see `examples/custom_searcher/local_search_runner`.
+
+## Files
+* **context_dir/asha.py**: The code for ASHA implemented as a custom SearchMethod.
+* **context_dir/run_experiment.py**: The code for running a custom SearchMethod locally with CoreSearchRunner.
+* **context_dir/model_def.py**: The core code for the model. This includes building and compiling the model.
+* **context_dir/data.py**: The data loading and preparation code for the model.
+* **context_dir/layers.py**: Defines the convolutional layers that the model uses. 
+
```

Comment:
It may be confusing to the reader/user to look at the files for both the search runner experiment and the multi-trial experiment alongside each other. Two ideas are:
1. Add a "Note: the context directory effectively contains files for two experiments: a multi-trial deep learning experiment and another experiment whose sole purpose is to wrap a CoreSearchRunner conducting a hyperparameter search within the first experiment." Or...
2. Find a way to put asha.py and run_experiment.py into a subdirectory.

Reply:
You're right, it will be confusing. To make separation clearer I moved multi-trial experiment files to subdirectory and updated README.md to indicate which files belong to searcher and which to the multi-trial experiment.